resend/mailExamples/StripeWelcomeEmail.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Hr,
+  Html,
+  Img,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "../utils/reactEmail.ts";
+
+export const StripeWelcomeEmail = () => (
+  <Html>
+    <Head />
+    <Preview>You're now ready to make live transactions with Stripe!</Preview>
+    <Body style={main}>
+      <Container style={container}>
+        <Section style={box}>
+          <Img
+            src={`https://react-email-demo-ndjnn09xj-resend.vercel.app/static/stripe-logo.png`}
```

Comment:
vercel? rsrsrsrsrsrsrsrsrs

Reply:
Is it a problem to maintain a vercel url? KKKKKKK